<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchResult</class>
 <widget class="QWidget" name="SearchResult">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="Result"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="back">
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="switchButton">
       <property name="text">
        <string>切换模式</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sort_bytime">
       <property name="text">
        <string>按时间排序</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sort_byprice">
       <property name="text">
        <string>按价格排序</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
